static/app/components/workflowEngine/gridCell/automationTitleCell.tsx

```diff
@@ -0,0 +1,25 @@
+import styled from '@emotion/styled';
+
+import Link from 'sentry/components/links/link';
+import {space} from 'sentry/styles/space';
+
+interface Props {
+  link: string;
```

Comment:
`href` would probably be a better prop name!

Reply:
fixed in https://github.com/getsentry/sentry/pull/91408/commits/18d5e17ffa8683a933830b66725d44282e127f3d